lib/spark/dsl/entity.ex

```diff
@@ -98,7 +98,7 @@ defmodule Spark.Dsl.Entity do
   @typedoc """
   A keyword list of nested entities.
   """
-  @type entities :: keyword(t)
+  @type entities :: keyword(t | [t])
```

Comment:
This seems strange, entities should be a keyword list no?

Reply:
You tell me, in the `entity nested definitions must be lists of entities or a single entity` test, we have this:
`entities: [child: @child, other_children: [@different_child]]`

So we for sure have a keyword list outside, but the value is apparently either one or more entities.

Reply:
This is something I changed in #218. It should also be fixed by #221 
I'm not sure why tests failed after merging, but not in the pull request...